client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -151,7 +150,7 @@ const Checkout = React.createClass( {
 			return false;
 		}
 
-		if ( this.props.transaction.step.name === transactionStepTypes.SUBMITTING_WPCOM_REQUEST ) {
+		if ( includes( [ transactionStepTypes.SUBMITTING_WPCOM_REQUEST, transactionStepTypes.RECEIVED_WPCOM_RESPONSE ], this.props.transaction.step.name ) ) {
```

Comment:
Would be nice to add a comment to explain why we don't want to redirect for those 2 steps.